content/get.md

```diff
@@ -67,82 +66,82 @@ CLM-Norway team we have to mainly two startingpoints:
 
 Change to fates directory and create your own branch to record all your
 changes
 
-    cd externals/fates
-    git checkout -b username_fates
+    [~/HOME/ctsm_fates_emerald/externals/cime]$ cd ../externals/fates
+    [~/HOME/ctsm_fates_emerald/externals/fates]$ git checkout -b username_fates
 
-If you do not create your own branch for \"cime\" and \"fates\", running
-\"./manage\_externals/checkout\_externals\", will overwrite your
-previous \"cime\" and \"fates\". You should be ready to create your
+If you do not create your own branch for `cime` and `fates`, running
+`./manage_externals/checkout_externals`, will overwrite your
+previous `cime` and `fates`. You should now be ready to create your
 first case.
 
 ### From the latest version of [CTSM](https://github.com/ESCOMP/CTSM)
 
 This tutorial assumes that you are logged into one of the clusters (fram
-or saga) at sigma2. For access to those see (future referance to
+or saga) at sigma2. For access to those see (future reference to
 prerequisites section).
 
 Start from your home folder and clone CTSM from ESCOMP 
 
-    git clone --origin escomp <https://github.com/ESCOMP/CTSM.git> CTSM
+    [~/HOME]$ git clone --origin escomp <https://github.com/ESCOMP/CTSM.git> CTSM
 
 Change into the new directory
 
-    cd CTSM
+    [~/HOME]$ cd CTSM
 
 Create a local branch 
 
-    git checkout master -b my_branch_name
+    [~/HOME/CTSM]$ git checkout master -b my_branch_name
 
 For later reference, it is useful to choose `my_branch_name` according
 to function and include the version and your username.
 
 To fetch the proper externals (CIME, FATES, etc.) run
 
-    ./manage_externals/checkout_externals
+    [~/HOME/CTSM]$ ./manage_externals/checkout_externals
 
 #### Porting of cime
 
-Now you need to add machine specifics for the norwegian clusters. This
+Now you need to add machine specifics for the Norwegian clusters. This
 can be done in two ways (check the
 [original](https://esmci.github.io/cime/versions/master/html/users_guide/porting-cime.html#steps-for-porting)
 documentation for a detailed explanation):
 
 1. You can replace some default configuration files with configuration files that contain details for these clusters.
-2. You can create a [.cime]{.title-ref} folder with the machine configurations under your home diretory.
+2. You can create a [.cime]{.title-ref} folder with the machine configurations under your home directory.
 
 
 ##### Replace the default configurations
 Execute the following steps: 
 1. Go into the cime-config folder
 
-        cd cime/config/cesm/machines 
+        [~/CTSM_ROOT]$ cd cime/config/cesm/machines 
     
 2. Delete the default files: 
 
-        rm config_machines.xml config_batch.xml config_compilers.xml
+        [~/CTSM_ROOT/cime/config/cesm/machine]$ rm config_machines.xml config_batch.xml config_compilers.xml
 
 3. Fetch replacementfiles from [this](https://github.com/gunnartl/config_files_sigma2.git) repository by typing
```

Comment:
"../gunnartl/..." this is bound to break sooner or later. 
Should not be hosted by a user but rather by our organization?

Reply:
Yes, I totally agree with the principle that we should use general repos instead of personal ones. 

I'm currently waiting for @gunnartl to join our team so I can open an issue assign/ask him about this.